### PR TITLE
Friendlier message when no bind addresses

### PR DESCRIPTION
**Pull request by huntc**
As this error always appears for first time users, we should present it in a friendlier fashion. This commit attempts to do that.

### CODE REVIEW

**Comment by huntc** [comment]
Sample output:

```
ChristohersMBP2:conductr-cli huntc$ sandbox run 2.0.0-rc.3 -n1:3 -f visualization -f logging -f monitoring
|------------------------------------------------|
| Starting ConductR                              |
|------------------------------------------------|
|------------------------------------------------|
| Whoops. Network address aliases are required   |
| so that the sandbox can operate as a cluster   |
| of machines                                    |
|------------------------------------------------|

Please run the following and then try your command again:

sudo ifconfig lo0 alias 192.168.10.1 255.255.255.0
sudo ifconfig lo0 alias 192.168.10.2 255.255.255.0
sudo ifconfig lo0 alias 192.168.10.3 255.255.255.0
```

File: conductr_cli/host.py [hunk 1]
```diff
@@ -81,10 +82,9 @@ def is_listening(ip_addr, port):
 
 def addr_alias_setup_instructions(addrs, subnet_mask):
     def info_text(commands):
-        if len(commands) > 1:
-            return 'Run the following commands to create the address aliases on your machine:'
-        else:
-            return 'Run the following command to create an address alias on your machine:'
+        return headline(
```
Comment (markusjura) [inline conductr_cli/host.py]: I would not use a headline here. Headlines should we used for certain sub sections to start ConductR, such as `Starting HAProxy`.

**Review by markusjura — CHANGES_REQUESTED** [review]

**Comment by markusjura** [comment]
Unit tests are failing due to not updated tests.

File: conductr_cli/host.py (hunk 1 above)
Comment (huntc) [reply conductr_cli/host.py]: Ok but I think it needs to standout somehow

**Comment by huntc** [comment]
I had problems running all tests due to the docker low on mem error throwing things out. I was being lazy. Apologies.

File: conductr_cli/host.py (hunk 1 above)
Comment (markusjura) [reply conductr_cli/host.py]: I am wondering if we should add the address aliases automatically after the user entered the sudo password. When prompting for the password, we could explain what we are doing. This way we wouldn’t need to highlight the message in a particular fashion.

What do you think?

**Comment by markusjura** [comment]
Fixed the unit tests. Also, the message `Whoops. Network address aliases are required so that the sandbox can operate as a cluster of machines.` is not marked as a headline anymore.

**New output**

```
|------------------------------------------------|
| Starting ConductR                              |
|------------------------------------------------|
Whoops. Network address aliases are required so that the sandbox can operate as a cluster of machines.

Please run the following and then try your command again:

sudo ifconfig lo0 alias 192.168.10.1 255.255.255.0
```

**Review by markusjura — APPROVED** [review]

**Pull request merged**